src/topology/category/Profinite.lean

```diff
@@ -187,24 +187,49 @@ has_limits_of_has_limits_creates_limits Profinite_to_Top
 instance has_colimits : limits.has_colimits Profinite :=
 has_colimits_of_reflective to_CompHaus
 
+universe variable u
```

Comment:
Can we use the `u` declared above instead of making a new declaration here? Or just add `universe u` nearer the top of the file

Reply:
Done